### ISSUE TITLE
Change links in recruitment emails from Lookit to CHS

### ISSUE THREAD

**Issue by msheskin** [lookit/lookit-api#1336]
# TL;DR
The recruitment emails sent out by lookit-bot (e.g., We're writing to invite you and your children CHILDNAME1 and CHILDNAME2 to participate in the study "Introduction to Project GARDEN!"!) include links that go to the lookit URL version of the study, rather than the CHS url, and this should be updated so links are CHS instead. 

# Narrative
As we are switching over from Lookit to CHS branding and default using the CHS domain, it is becoming out of date for recruitment emails to go to the Lookit domain. This is a particular pain point for Project GARDEN, because the exit URLs for GARDEN studies go to the CHS version of our lab page. Importantly, there is no solution that will work elegantly for all families and use cases, and so we might also want to add some explanatory text to the recruitment emails. Specifically, many families may have ONLY visited the Lookit domain, and when they are directed to the newer CHS domain for the first time they will not be logged in and their login credentials will not be saved in their browser. 

# Implementation Notes
Here is a proposal. Take the link off of the study name, and instead put both versions of the link after the introductory sentence, as in this example:

We're writing to invite you and your children CHILD1 and CHILD2 to participate in the study "Introduction to Project GARDEN!"! This study is run by the Project GARDEN at The University of Texas at Dallas.

Click [here](https://lookit.mit.edu/studies/7696675f-3a00-4ab7-a273-a1129318c5e8/) to find this study on our older Lookit website
Click [here](https://childrenhelpingscience.com/studies/7696675f-3a00-4ab7-a273-a1129318c5e8/) to find this study on our newer Children Helping Science website

**Comment by becky-gilbert** [comment]
This change relies on the value of a configuration variable (`BASE_URL`) that is set in the orchestrator repo. So this change to the link domains in emails will be happen when we merge the changes in orchestrator related to redirecting from MIT to CHS domains, and making CHS the primary domain: https://github.com/lookit/lookit-orchestrator/pull/49 
After merging that orchestrator PR, we will need to rebuild/deploy the production site to see the changes. I will close this issue when this change has actually gone into production.

**Comment by becky-gilbert** [comment]
Closed via #1366
